test/jasmine/tests/dragelement_test.js

```diff
@@ -122,6 +122,18 @@ describe('dragElement', function() {
         expect(countCoverSlip()).toEqual(0);
     });
 
+    it('should not add a cover slip div to the DOM when right click', function() {
+        function countCoverSlip() {
+            return d3.selectAll('.dragcover').size();
+        }
+
+        var options = { element: this.element, gd: this.gd };
+        dragElement.init(options);
+
+        mouseEvent('mousedown', this.x, this.y, { buttons: 2 });
+        expect(countCoverSlip()).toEqual(0);
```

Comment:
Great! Very clear correspondence to the test above. Two last changes I would suggest, then I think this is ready to go:
- 🌴  Can you move `countCoverSlip` out one scope so we don't have to repeat it?
- Can you continue through `mouseup` as in the test below, and verify that `handleClick` has not been called? (that one ironically looks annoying to DRY 🌴  so you can just repeat the pieces you need.)

Reply:
Sure